Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/Internal/Rectangle.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzQtComponents/Components/Widgets/Internal/Rectangle.h>
+
+#include <QPaintEvent>
+#include <QPainter>
+#include <QWidget>
+
+namespace AzQtComponents::Internal
+{
+    Rectangle::Rectangle(QWidget* parent)
+        : QWidget(parent)
+    {
+    }
+    void Rectangle::setColor(const QColor& color)
```

Comment:
uber nit: Could we add a break after the ending curly brace to leave a space between all member functions defined in the .cpp

Reply:
(repeated below, + missing EOL character)

Reply:
I'll add the blank lines.

I also found out, that since clang-format 14 there's a directive `SeparateDefinitionBlocks: Always` and it adds these breaks when formatting the file. Probably it makes sense to add it. I can do it in separate patch.

Reply:
version 14 is very new but probably, older version will just ignore this option when it's encountered.

Reply:
Hmm, VS2022 shows error dialogs when it encounters uknown setting in `.clang-format`. Better not do it then.

Reply:
Anyway. Done.

Reply:
Awesome thank you! (we did at one point provide our own version of clang-format which could be the latest one but this got complicated to maintain so now we just default to whatever is the minimum version supported in Visual Studio).